.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
```

Comment:
this is so it works with all code files in the project and in case if we moved to use .ts in the future we will be consistent + this the same file I grabbed from the client project.